app/views/content_assets/_form.html.erb

```diff
@@ -23,8 +23,8 @@
 <div class="govuk-grid-column-one-half">
   <%= form.submit('Submit', :class => "govuk-button") %>
 </div>
-</div>   
+</div>
 
 <% end %>
 
-<%= link_to 'GOV.UK image upload guidelines', 'https://design-system.service.gov.uk/styles/images/', target: "_blank" %> | 
\ No newline at end of file
+<%= link_to 'GOV.UK image upload guidelines', 'https://design-system.service.gov.uk/styles/images/', target: "_blank" %> |
```

Comment:
When the index shows lots of rows, the add button disappears below the fold, so I've duplicated above the list of assets